test/test_map.py

```diff
@@ -36,6 +36,7 @@
 class TestMap(unittest.TestCase):
     def setUp(self):
         np.random.seed(1234)
+        bf.map_clear_cache()
```

Comment:
Everything in `test_map` and `test_transpose` bombed because `map_clear_cache` is not exported properly:
```
ERROR: test_complex_float (test_map.TestMap)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/qblocks/bifrost/test/test_map.py", line 39, in setUp
    bf.map_clear_cache()
AttributeError: module 'bifrost' has no attribute 'map_clear_cache'
```
I can see `def clear_cache` in `bifrost/map.py` but it's not exported by `bifrost/__init__.py`

Reply:
Ah, yes, a last minute change in the naming inside `bifrost.map`.  I didn't really like `map_clear_cache` so I went with a simpler `clear_cache`.  Maybe a `clear_map_cache` and `list_map_cache` would be better?  That seems a little more readable once they are imported into `bifrost/__init__.py` than a simple `clear_cache`.  Clear what cache?

Reply:
Yep. As long as we're keeping flat-ish namespace, `bifrost.clear_map_cache` (and `list_map_cache`) has my vote. If we wanted to get hierarchical, then `bifrost.map.clear_cache` but that doesn't seem worthwhile.